### PR TITLE
Opsexp 1293 imagemagick Imagemagick-7.1.0-16 version update

### PR DESCRIPTION
**Pull request by slohe1**
Ref: OPSEXP-1293

### CODE REVIEW

**Review by alxgomz — APPROVED** [review]
OK but we need more work to allow deploying the right IM version for the right ACS version. TBD in another task

File: roles/transformers/vars/Ubuntu20.yml [hunk 1]
```diff
@@ -1,2 +1,4 @@
 ---
-imagemagick_version: 7.0.10-59
+imagemagick_version: 7.1.0-16-ci
+imagemagick_packages:
+  - imagemagick-distribution-ubuntu-20.04
```
Comment (gionn) [inline roles/transformers/vars/Ubuntu20.yml:4]: leave a blank line at the end of each file (diff should not show the ⛔ )

**Review by gionn — CHANGES_REQUESTED** [review]

File: roles/transformers/vars/Ubuntu20.yml (hunk 1 above)
Comment (alxgomz) [reply roles/transformers/vars/Ubuntu20.yml:4]: my linter usually complains about empty blank lines at the end of the yaml files.
Does yours complain about the opposite?
We need to set that in our ways of working. Any preferences gyus?

Comment (slohe1) [reply roles/transformers/vars/Ubuntu20.yml]: For me it is working in both the ways. 
Currently for this added a blank line at the end.

**Review by gionn — CHANGES_REQUESTED** [review]
if version is always the same, now it can be moved to the group_vars/all.yml

remove from Debian.yml and RedHat.yml the now unused imagemagick_packages

File: roles/transformers/vars/Ubuntu20.yml (hunk 1 above)
Comment (gionn) [reply roles/transformers/vars/Ubuntu20.yml]: will address this with https://github.com/Alfresco/alfresco-ansible-deployment/pull/456

Comment (alxgomz) [reply roles/transformers/vars/Ubuntu20.yml]: 🏅

**Comment by alxgomz** [comment]
@gionn this is dealt with in OPSEXP-1744

**Review by gionn — APPROVED** [review]

**Pull request merged**